time/src/main/java/net/jqwik/time/internal/properties/arbitraries/valueRanges/ValueRange.java

```diff
@@ -0,0 +1,61 @@
+package net.jqwik.time.internal.properties.arbitraries.valueRanges;
+
+public abstract class ValueRange<T> {
+
+	private T min;
+	private T max;
+
+	protected void exceptionCheck(Parameter parameter) {
```

Comment:
"exceptionCheck" is a name that suggests that an exception is being checked. What you really do is checking the range's validity. Suggested name: checkRangeIsValid or checkValidity or st like that.